{
    "id": "problem_85",
    "description": "\n\nWhen attempting to create a new dictionary, a TypeError is raised stating that the input is not a valid type for a dictionary. Specifically, the error states that the input is a dict type, when it should be a string, list, tuple, or set.",
    "code": "dict_example = {1: 'a', 2: 'b'}\nnew_dict = dict(dict_example)",
    "tags": [
        "TypeError",
        "dict",
        "Hard"
    ]
}